explorer/src/components/Nav.tsx

```diff
@@ -112,51 +111,67 @@ const navOptions: navOptionType[] = [
   {
     url: '/overview',
     title: 'Overview',
-    Icon: <BarChart />,
+    renderIcon: (mode) => <OverviewSVG mode={mode} />
   },
   {
     url: '/network-components',
     title: 'Network Components',
-    Icon: <ConnectedTv />,
+    renderIcon: (mode) => <NetworkComponentsSVG mode={mode} />,
     nested: [
       {
         url: '/network-components/mixnodes',
         title: 'Mixnodes',
-        Icon: <TrafficSharp />,
       },
       {
         url: '/network-components/gateways',
         title: 'Gateways',
-        Icon: <GarageTwoTone />,
+      },
+      {
+        url: `${BIG_DIPPER}/validators`,
+        title: 'Validators',
       },
     ],
   },
   {
     url: '/nodemap',
     title: 'Nodemap',
-    Icon: <Pin />,
+    renderIcon: (mode) => <NodemapSVG mode={mode} />,
   },
 ];
 
 const ExpandableButton: React.FC<navOptionType> = ({
   nested,
   title,
-  Icon,
+  renderIcon,
   url,
 }) => {
   const [open, toggle] = React.useState(false);
+  const { mode } = React.useContext(MainContext);
```

Comment:
the svg icons can get this directly so no need to access here and pass as a prop